virtcontainers/clh.go

```diff
@@ -425,8 +432,62 @@ func (clh *cloudHypervisor) hypervisorConfig() HypervisorConfig {
 }
 
 func (clh *cloudHypervisor) resizeMemory(reqMemMB uint32, memoryBlockSizeMB uint32, probe bool) (uint32, memoryDevice, error) {
-	clh.Logger().WithField("function", "resizeMemory").Warn("not supported")
-	return 0, memoryDevice{}, nil
+	memDev := memoryDevice{probe: probe}
```

Comment:
In general, I feel we should always return `memoryDevice{}` in this function. If I read correctly the`resizeMemory()` from `virtcontainers/qemu.go`, they will need to return a non-empty `memoryDevice` only when `hotplugMemory` is required (e.g. removing/adding memory device when `virtio-mem` is not used). I am assuming we don't need to (manually) remove/add memory device with `clh` for memory hotplug.

Reply:
Good catch, yeah it is not needed I added a validation to check if probe is true and return an error.